.github/assistant.py

```diff
@@ -59,7 +59,7 @@ def prune_packages(req_file: str, *pkgs: str) -> None:
             lines = [ln for ln in lines if not ln.startswith(pkg)]
         logging.info(lines)
 
-        with open(req_file, "w") as fp:
+        with open(req_file, "w", encoding="utf_8") as fp:
```

Comment:
```suggestion
        with open(req_file, "w", encoding="utf-8") as fp:
```

as done by the docs here: https://docs.python.org/3/howto/unicode.html